app/models/loader/orchestrate.rb

```diff
@@ -118,6 +118,67 @@ def load
       store_passwords
 
       store_public_keys
+
+      emit_audit
+    end
+
+    SDID = Audit::SDID
```

Comment:
Some of these names are really cryptic. As a casual observer of the code I don't want to have to read the RFC to understand what this means. I would like to see some comments indicating what the variable and constant names mean and how they are used.

Reply:
Fair points.